src/ArcherContainer.js

```diff
@@ -2,18 +2,27 @@
 
 import React from 'react';
 import ResizeObserver from 'resize-observer-polyfill';
-import Point from './Point';
 
+import Point from './Point';
 import SvgArrow from './SvgArrow';
 
+export type ArcherContainerContextType = {
+  registerChild: (string, HTMLElement) => void,
+  registerTransitions: (string, Array<SourceToTargetType>) => void,
+  unregisterChild: string => void,
+  unregisterTransitions: string => void,
+} | null;
+
+type FunctionChild = (context: React$Context<ArcherContainerContextType>) => React$Node;
+
 type Props = {
   arrowLength: number,
   arrowThickness: number,
   strokeColor: string,
   strokeWidth: number,
   strokeDasharray?: string,
   noCurves?: boolean,
-  children: React$Node,
+  children: React$Node | FunctionChild,
```

Comment:
The typescript definitions would need an updated as well with this change :blush:

Reply:
By the way we don't specifically type the `children` prop, so that part is fine.